samples/index.html

```diff
@@ -38,10 +38,10 @@ <h2>Sample</h2>
 			React.createElement( CKEditor, {
 				editor: ClassicEditor,
 				data: editorContent,
-				onInit: editor => {
+				onReady: ( { editor } ) => {
```

Comment:
I have a strong feeling that we should not modify the API of the `onInit`/`onReady` hook.

If the callback is called with an editor instance only, I guess we can use the current (v1) API.